packages/localstorage/examples/localstorage/pages/demo/demo.page.ts

```diff
@@ -0,0 +1,93 @@
+import { Component } from '@angular/core';
+import { LocalstorageService } from '@acpaas-ui/ngx-components/localstorage';
+
+@Component({
+	templateUrl: './demo.page.html',
+	providers: [ LocalstorageService ],
+})
+export class LocalstorageDemoPageComponent {
+
+	public user: any;
+	public item: any;
+
+	constructor(
+		private localstorageService: LocalstorageService
+	) {
+		this.user = this.localstorageService.select('user');
+	}
+
+	public typescript1 = `import { LocalstorageModule } from '@acpaas-ui/localstorage';
```

Comment:
`ngx-components`
And for readability it's probably better to bundle all `public` modifiers, before writing the rest of the code.